src/sdks/tableau/apis/pulseApi.ts

```diff
@@ -26,9 +26,18 @@ const listAllPulseMetricDefinitionsRestEndpoint = makeEndpoint({
         - 'DEFINITION_VIEW_FULL' - Return the metric definition and the specified number of metrics.
         - 'DEFINITION_VIEW_DEFAULT' - Return the metric definition and the default metric.`,
     },
+    {
+      name: 'page_token',
+      type: 'Query',
+      schema: z.optional(z.string()),
+      description: 'Token for retrieving the next page of results. Omit for the first page.',
+    },
   ],
   response: z.object({
     definitions: z.array(pulseMetricDefinitionSchema),
+    next_page_token: z.string().optional(),
+    offset: z.coerce.number(),
+    total_available: z.coerce.number(),
```

Comment:
There's also a `page_size` param I think. The other tools that support pagination also include a `pageSize` argument. Was it intentionally omitted? I think we should probably include it so folks who want 1000 results don't have to call the API 20 times with a default page size of 50 (I'm assuming it's 50 but have no idea)